watson/cli.py

```diff
@@ -833,8 +835,8 @@ def remove(watson, id, force):
 
     if not force:
         click.confirm(
-            "You are about to remove frame "
-            "{project}{tags_spacer}{tags} from {start} to {stop}, continue?".format(
+            'You are about to remove frame {project}{tags_spacer}{tags} from '
```

Comment:
Thanks for your effort in unifying single/double quotes, but for messages shown to the user, which are (in theory) translatable, we prefer double quotes. This usage might not be consistent in the existing code, but for new code please adhere to this convention.

Reply:
Fixed in https://github.com/TailorDev/Watson/pull/179/commits/903e75fafdbd51b8ea2f04f087fcd2d0e1f13534 (part of #179)